src/smart-components/role/add-role-new/add-permissions.js

```diff
@@ -182,7 +183,21 @@ const AddPermissionsTable = ({ selectedPermissions, setSelectedPermissions, ...p
       return;
     }
 
-    const basePermissions = baseRole?.access || [];
+    let notAllowed = [];
+
+    const basePermissions =
+      baseRole?.access.filter((item) => {
+        if (applicationOptions.includes(item?.permission?.split(':')[0])) {
+          return true;
+        } else {
+          notAllowed.push(item);
+        }
+        return false;
```

Comment:
No need for the `else` statement.
```suggestion
        }
        
        notAllowed.push(item);
        return false;
```